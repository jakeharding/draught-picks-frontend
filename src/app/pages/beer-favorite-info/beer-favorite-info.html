<!--
  Generated template for the GoToBeerFavoriteInfoPage page.

  See http://ionicframework.com/docs/components/#navigation for more info on
  Ionic pages and navigation.
-->
<ion-header>
  <draught-header></draught-header>
</ion-header>

<ion-content>
  <ion-row>
    <ion-col class="below-header ion-text-center"
             size="12" sizeSm="10" offsetSm="1" sizeMd="8" offsetMd="2" sizeLg="6" offsetLg="3" sizeXl="4" offsetXl="4">

      <h1>Description of Favorite Beer(s) Tips</h1>

      <p class="ion-text-left ion-margin"> Please describe positive facts about your favorite beers rather than negative facts to get the best results!</p>

      <ion-row>
        <ion-col>
          <ion-row>
            <ion-col class="ion-text-left">
              <p><b>Good Example:</b><br></p>
              <p>"I enjoy the taste of Bud Light and sour lagers."</p>
            </ion-col>
          </ion-row>

          <ion-row>
            <ion-col class="ion-text-left">
              <p><b>Bad Example:</b><br></p>
              <p>"I do not like sweet and light beers. "</p>
            </ion-col>
          </ion-row>
        </ion-col>
      </ion-row>
      <br>
      <a class="backToPreferences" routerLink="/tabs/preferences" routerDirection="back">Back To Your Preferences</a>
    </ion-col>
  </ion-row>
</ion-content>
