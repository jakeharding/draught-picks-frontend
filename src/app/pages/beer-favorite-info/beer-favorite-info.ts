import { Component } from '@angular/core';
// import { IonicPage } from '@ionic/angular';
import { BasePage } from '../BasePage';

/**
 * Generated class for the BeerFavoriteInfoPage page.
 *
 * See https://ionicframework.com/docs/components/#navigation for more info on
 * Ionic pages and navigation.
 */
// @IonicPage()
@Component({
  selector: 'page-go-to-beer-favorite-info',
  templateUrl: 'beer-favorite-info.html',
})
export class BeerFavoriteInfoPage extends BasePage {

  /**
   * BeerFavoriteInfoPage constructor
   * empty but required constructor for the BeerFavoriteInfoPage Class, does nothing
   */
  constructor() {
    super('beer-favorite-info');
  }

}
