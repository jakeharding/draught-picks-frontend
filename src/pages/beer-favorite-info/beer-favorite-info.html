<!--
  Generated template for the GoToBeerFavoriteInfoPage page.

  See http://ionicframework.com/docs/components/#navigation for more info on
  Ionic pages and navigation.
-->
<draught-header></draught-header>

<ion-content>
  <ion-row>
    <ion-col class="below-header" text-center col-12 col-sm-10 offset-sm-1 col-md-8 offset-md-2 col-lg-6 offset-lg-3 col-xl-4 offset-xl-4>
      <h1>Description of Favorite Beer(s) Tips</h1>

      <p text-left margin> Please describe positive facts about your favorite beers rather than negative facts to get the best results!
        <p>
            <li>Good Example: "I enjoy the taste of Bud Light and sour loggers."</li>
            <li>Bad Example: "I do not like sweet and light beers. "</li>
        </p>
    </p>

      <a navPop>Done</a>
    </ion-col>
  </ion-row>
</ion-content>
