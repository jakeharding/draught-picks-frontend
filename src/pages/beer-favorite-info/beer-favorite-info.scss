page-go-to-beer-favorite-info {

}
