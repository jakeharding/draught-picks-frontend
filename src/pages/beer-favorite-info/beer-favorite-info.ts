import { Component } from '@angular/core';
import { IonicPage, NavController, NavParams } from 'ionic-angular';
import { BasePage } from "../BasePage";

/**
 * Generated class for the BeerFavoriteInfoPage page.
 *
 * See https://ionicframework.com/docs/components/#navigation for more info on
 * Ionic pages and navigation.
 */
@IonicPage()
@Component({
  selector: 'page-go-to-beer-favorite-info',
  templateUrl: 'beer-favorite-info.html',
})
export class BeerFavoriteInfoPage extends BasePage {

  /**
   * BeerFavoriteInfoPage constructor
   * empty but required constructor for the BeerFavoriteInfoPage Class, does nothing
   * */
  constructor(public navCtrl: NavController, public navParams: NavParams) {
    super('beer-favorite-info');
  }

}
