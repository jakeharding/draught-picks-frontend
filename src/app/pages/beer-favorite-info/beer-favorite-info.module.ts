import { NgModule } from '@angular/core';
import { IonicModule } from '@ionic/angular';
import { BeerFavoriteInfoPage } from './beer-favorite-info';
import { ComponentsModule } from '../../components/components.module';
import { RouterModule } from '@angular/router';


@NgModule({
  declarations: [
    BeerFavoriteInfoPage,
  ],
  imports: [
    IonicModule,
    RouterModule.forChild([{path: '', component: BeerFavoriteInfoPage}]),
    ComponentsModule,
  ],
})
export class BeerFavoriteInfoPageModule {}
