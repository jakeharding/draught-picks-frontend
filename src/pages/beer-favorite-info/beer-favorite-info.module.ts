import { NgModule } from '@angular/core';
import { IonicPageModule } from 'ionic-angular';
import { BeerFavoriteInfoPage } from './beer-favorite-info';
import { ComponentsModule } from '../../components/components.module';


@NgModule({
  declarations: [
    BeerFavoriteInfoPage,
  ],
  imports: [
    IonicPageModule.forChild(BeerFavoriteInfoPage),
    ComponentsModule,
  ],
})
export class BeerFavoriteInfoPageModule {}
