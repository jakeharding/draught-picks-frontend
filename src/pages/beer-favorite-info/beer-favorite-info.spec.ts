import { async, ComponentFixture, TestBed } from '@angular/core/testing';
import { BeerFavoriteInfoPage } from './beer-favorite-info';
import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
import { NavController } from 'ionic-angular';

/**
 * beer-favorite-info.spec.ts
 *
 * Created by jake
 * Created on 11/18/18
 * 
 * Test the BeerFavoriteInfoPage.
 */

describe('BeerFavoriteInfoPage', () => {
  let fixture: ComponentFixture<BeerFavoriteInfoPage>;
  let instance: BeerFavoriteInfoPage;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      schemas: [CUSTOM_ELEMENTS_SCHEMA],
      declarations: [BeerFavoriteInfoPage],
      providers: [
        { provide: NavController, useValue: {}}
      ]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(BeerFavoriteInfoPage);
    instance = fixture.debugElement.componentInstance;
  });

  it('should initialize the BeerFavoriteInfoPage', () => {
    expect(instance).toBeTruthy();
    expect(fixture).toMatchSnapshot();
  });
});
