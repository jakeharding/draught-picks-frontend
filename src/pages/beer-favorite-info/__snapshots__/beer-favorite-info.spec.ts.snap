// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeerFavoriteInfoPage should initialize the BeerFavoriteInfoPage 1`] = `
<page-go-to-beer-favorite-info>
  
  <draught-header />
  
  <ion-content>
    
    
    <ion-row>
      
      
      <ion-col
        class="below-header"
        col-12=""
        col-lg-6=""
        col-md-8=""
        col-sm-10=""
        col-xl-4=""
        offset-lg-3=""
        offset-md-2=""
        offset-sm-1=""
        offset-xl-4=""
        text-center=""
      >
        
        
        <h1>
          Description of Favorite Beer(s) Tips
        </h1>
        
  
        
        <p
          margin=""
          text-left=""
        >
           Please describe positive facts about your favorite beers rather than negative facts to get the best results!
        </p>
        
  
        
        <ion-row>
          
          
          <ion-col>
            
            
            <ion-row>
              
              
              <ion-col
                text-left=""
              >
                
                
                <p>
                  <b>
                    Good Example:
                  </b>
                  <br />
                </p>
                
                
                <p>
                  "I enjoy the taste of Bud Light and sour lagers."
                </p>
                
              
              </ion-col>
              
            
            </ion-row>
            
  
            
            <ion-row>
              
              
              <ion-col
                text-left=""
              >
                
                
                <p>
                  <b>
                    Bad Example:
                  </b>
                  <br />
                </p>
                
                
                <p>
                  "I do not like sweet and light beers. "
                </p>
                
              
              </ion-col>
              
            
            </ion-row>
            
          
          </ion-col>
          
        
        </ion-row>
        
        
        <br />
        
        
        <a
          class="backToPreferences"
          navpop=""
        >
          Back To Your Preferences
        </a>
        
      
      </ion-col>
      
    
    </ion-row>
    
  
  </ion-content>
  
</page-go-to-beer-favorite-info>
`;
